src/EntityListener/RevokeReferentTeamMemberRolesListener.php

```diff
@@ -49,6 +50,8 @@ public function preFlush(Adherent $adherent): void
             $personLink->setIsMunicipalManagerSupervisor(false);
         }
 
+        $this->entityManager->getRepository(DelegatedAccess::class)->removeFromDelegator($adherent);
```

Comment:
we should handle this change in `src/EntityListener/RevokeDelegatedAccessListener.php` too as this listener is part of the old system and will be removed

Reply:
updated